exchanges/btcmarkets/btcmarkets.go

```diff
@@ -463,21 +616,3 @@ func (b *BTCMarkets) GetFee(feeBuilder *exchange.FeeBuilder) (float64, error) {
 func getOfflineTradeFee(price, amount float64) float64 {
```

Comment:
With the change to maker/taker on the crypto/crypto market the worst possible fee for them is no longer a flat 0.0085

Reply:
https://www.btcmarkets.net/fees
dont we have to assume the worst case scenario? and also getofflinetradnig fees doesnt have to be crypto/crypto does it?

Reply:
Yes and the worst case scenario is not 0.85% anymore for btc/<crypto> trades this will cause a miscalculation of fee's